apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -292,6 +291,7 @@ export default function ListBoxInline({ options = {} }) {
             {({ height, width }) => (
               <ListBox
```

Comment:
Should we apply React.memo(ListBox) instead of React.memo(ListBoxInline)

Reply:
maybe (or memo both)
but doing a working memo on ListVox would require more work as it has a lot of properties that would need to be stable